### ISSUE TITLE
Find a way to get rid of scrollbars

### ISSUE THREAD

**Issue by neumanrq** [Absolventa/iruby-chartkick#2]
<img width="919" alt="Bildschirmfoto 2019-11-03 um 09 34 58" src="https://user-images.githubusercontent.com/3685123/68082452-449a9980-fe1d-11e9-9aa6-ceb6ee68f6f7.png">

**Comment by axlwaii** [comment]
Seems like you need to include some styles:
either inline:
```html
<div style="overflow: hidden;">
    <!--- The chart -->
</div>
```

or as class
```css
.char-class-name {
  overflow: hidden;
}
```

**Comment by neumanrq** [comment]
Good catch, @axlwaii! We could try that. Maybe we can also add a keyword argument controlling that, like

```Ruby
line_chart(scrollbars: true) # it's false by default
```

**Comment by elalemanyo** [comment]
Maybe helps: https://github.com/ankane/chartkick/issues/194

**Comment by elalemanyo** [comment]
or maybe we can just add some `width` and `height` options?
https://github.com/ankane/chartkick#options
